doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
-   > * CNTT/OPNFV is NOT planning to provide a production compute solution for RI
+•	ETSI NFV does not have a separation between HW Infrastructure Management and SW Virtualisation Management and by that the cardinality of having multiple CaaS and IaaS layers on top of a shared HW Infrastructure Layer cannot be expressed
 
+•	ETSI NFV lacks a description of the reference points in between the SW Virtualisation Layers and the HW Infrastructure Layer denoted as Vl-Ha and by that cannot express Packet Flows, Control/Status Interfaces and Management Interfaces between them
 
-<a name="5.1.1"></a>
-### 5.1.1 Executive Summary
+•	ETSI NFV model does not explicitly call out an HW Abstraction layer on top of the HW resources and by that many SW Virtualisation Layers act upon the physical HW components in a non-portable way that do not allow an HW Infrastructure to compose abstracted HW resources from pools of individual HW components
+
+•	ETSI NFV lack of spelled out HW Abstraction does not enable a clear way to abstract and describe the technical possibilities that HW layer networking separation, encapsulation, acceleration, etc. can be implemented either in a HW Infrastructure Managed SmartNIC on the Server units or in the shared underlay switching fabric i.e. on the physical Switching Units 
+
+•	ETSI NFV model does not include any reference to SDN (Software Define Network) controller(s) and relevant integration points into NFVI and MANO 
+
+•	ETSI NFV model does not have a way to enable programmable forwarding planes in the HW layer controlled from higher layers of virtualization managers, orchestrators or Network Functions
+
+<p align="center"><img src="../tech/figures/ETSI.png"><b> Figure 5-1-1-1: ETSI model and Networking.</b></p>
+
+Note: Other areas of concern may be identified and addressed during the later stages of work on the RM/RA contents for Networking.
+
+<a name="5.1.1.2"></a>
+#### 5.1.1.2 Specific topics to be included in the Networking Scope
+To address the concern areas detailed above the specific topics listed below will be covered under Networking. The list below does not pretend to be complete. It will be a subject to modifications as a result of the community discussion.
+
+•	Network connectivity for L3 tenants, GWs, SDS, etc
+
+•	Spine-Leaf Architecture ( 3 Tier or 5 Tier CLOS architecture )
+
+•	Automation of the programmable newtorking fabric
+
+•	Variety of Underlay protocols that can be used as a transport layer within the physical fabric
+
+•	Variety of Overlay protocols
+
+•	SDN integration with Infrastructure Management
+
+•	SRIOV workload interworking
+
+•	DPDK workload interworking
+
+•	Service Function Chaining (SFC ) and how it can be achieved
+
+•	New networking realisation technologies ( SmartNIC , FPGA ,..)
+
+•	Inter K8s POD connectivity and CNI
+
+•	Networking aspects of migration from IaaS to CaaS (from the VNF paradigm to the CNF paradigm)
+
+•	Coexistence of IaaS and CaaS and protracted parallel operations
```

Comment:
Maybe in addition to co-existence of IaaS and CaaS infrastructures, we need to investigate networking aspects of interworking between hypervisor-based workloads and containerized workloads.
According to ETSI, a VNF can contain hypervisor-based workloads and containerized workloads in parallel. Maybe not every infrastructure will support this, but ETSI standard already includes it.

Reply:
@ulikleber I think this is a tricky questions, and just by that statement, probably needs our attension both in terminology chapeters and CNTT deployment guidelines and rules.
On the one hand a VNF (as in Hypervisor based workload) and a CNF (as in Container based workload) should be able to communicate with each other, but that should be on the rather high level Network Function interfaces/protocols as an IMS system communicate with a Packet Gateway. This would not create any major complexity since it is business as usual.
On another hand I have understood ETSI NFV to specify NFVI to span all infrastructure even on geographically distributed sites, thereby allowing a VNF (in the ETSI NFV terminology you used it in above) to be geographically distributed, but I would not recomend that myself and it brings considerable complexity. Is that is a wanted scenarion for CNTT to cover?

Reply:
@TFredberg we discussed few times regarding geographically distributed VNFs but i do not think that the current RM and RA1 took this use case into the consideration.  It is worth checking though. 
@ulikleber thanks for confirmation that ETSI supports co-existence of IaaS and Caas infrastructures (which I did not doubt).  Can you please help us and direct to the relevant document(s)?  I would strongly insist that we should define our structures in a way that conforms to the ETSI view on this as much as possible. Will appreciate.

Reply:
Answering @TFredberg : I have to disagree. If a VNF contains a containerized and hypervisor-based workloads, the internal connection between those is inside the VNF and not on NS level. Therefore if we concentrate on splitting CaaS and IaaS in the NFVI, we have to talk about such interfaces explicitly.
Answering @walterkozlowski : ETSI as it is an SDO will not define product requirement such as xxx shall support a and b in parallel. ETSI NFV documents consider containers a virtualization technology and in most places tries to stay independent. The work in IFA029 shows the principles how container-specifics that had not been considered before will be added. IFA040 (currently in final review) adds new aspects for the modelling needed in MANO to manage containerized workloades. Work is progressing now to specify requirements on NFVO, VNFM, VIM, CISM and CIR to better reflect containerized VNFs. Documents like IFA010 (overall functional requirements), etc. specify how to manage and orchestrate VNFs - both containerized and hypervisor-based. By defining interfaces and models in a way that allows parallel use of different technologies, a product will be able to support that. But I doubt that ETSI will include explicite statements on parallel support. For us in CNTT that could mean to define a RA/RI, that supports container only and another RA/RI that supports VM only, and in addition a RA/RI the provides parallel support.

Reply:
@ulikleber I think you bring up a third usage case "mixed containerized and hypervisor-based workloads" that I didn´t cover in my "one hand and another hand" examples.
I would suggest that these should be out of CNTT scope since I think it is highly unlikely that some applications will implement a mixed of containerized and hypervisor-based workloads that are expected to be deployed onto a set of Cloud Infrastructure deployed IaaS and CaaS environments. This would create a rather strange and very complex mix and coordination requirements of Operator Management especially of Lice Cycle Management in these multiple domains.
I could though expect to see applications that bring their internal CaaS on which the containerized workloads run as a VM on the Cloud Infrastructure provided IaaS.

Reply:
I think that we should not introduce restrictions that are not in the standard. But let's concentrate on the bullet above. What kind of Coexistence is meant here? I think it should refer to networking of containerized and hypervisor-based workloads, not be separating them to different implementations of cloud infrastructure, but support interworking.